### DISCUSSION TITLE
[ui-base] allow customizable textSeasonEpisode

### DISCUSSION THREAD

**Discussion by hrk** [trakt/showly#572] [Ideas / Feature Requests]
Hello, I'm new here so please forgive me if I'm posting this in the wrong place. From the main readme, it looked like this was more fit for discussion but, if not the case, educate me if a "issue" should have been a better way. 

Current notifications for new episodes have a fixed format, taken from strings.xml: <string name="textSeasonEpisode">S.%02d E.%02d</string>

Would it be possible to have it customizable either choosing from a set of pre-made formats or a totally customizable one using variables (maybe overkill)? Examples of pre-made formats that are often used:
- 1x01
- S1:E1
- s1:e1
- S01E01
- s01e01

The above examples come from a similar/competing open-source app offering the "Number format" option in its settings.

If there is interest in providing this feature, I'm willing to propose a PR. However, since I've been away from android development for some years, I'd appreciate some pointers. At a glance, settings are provided in ui-settings module, while this string is used in ui-base module. How do they talk to each other?

**Comment by michaldrabik** [comment]
It's a good place to post the ideas : ) 
I had this idea before and will most probably introduce this in the app sooner or later.

Kind regards
